e2e/stack.go

```diff
@@ -41,16 +43,19 @@ type EventListener interface {
 }
 
 type StackConfig struct {
-	L1URL *url.URL
-	*rollup.Config
-	Operator L1User
-	Users    []L1User
+	L1URL         *e2eurl.URL
+	RUConfig      *rollup.Config
+	Operator      L1User
+	Users         []L1User
+	L1Client      *L1Client
+	L2Client      *bftclient.HTTP
+	MonomerClient *MonomerClient
```

Comment:
Can remove the `L1URL` and `RUConfig`?

Reply:
`L1URL` is removed here: 22dd6db

`RUConfig` has data still used to construct the op `portal` contract bindings. I do intend to swallow that up into `stack.go`, as the client constructions were. After that, the `RUConfig` can be removed as well.

Include in this PR? Or follow-up?

Reply:
follow-up sounds good, thanks :+1: